features/auth/signup/BasicForm.tsx

```diff
@@ -116,6 +102,26 @@ export default function BasicForm({
           helperText={errors?.email?.message ?? " "}
           error={!!errors?.email?.message}
         />
+        <InputLabel className={authClasses.formLabel} htmlFor="password">
+          Password
```

Comment:
```suggestion
          {t("auth:basic_form.password.field_label"}}
```
Since this is still hardcoding "Password"